docs/index.html

```diff
@@ -46,6 +46,11 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Supports code in production, even if they did not write it.</li>
         <li>Can feel like an impostor at times.</li>
         <li>Makes sure discussions are always productive and everyone gets to have their say before the team makes a decision. Only disregards opinions by providing arguments.</li>
+        <li>Create community and share your knowledge</li>
```

Comment:
Just need to add a period on each of the lines (will create suggestions that you can automatically merge inside of GitHub):

```suggestion
        <li>Create community and share your knowledge.</li>
```